ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -501,6 +502,12 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         document.querySelector('.video-js-parent')?.append(window.oldSavedDiv);
       }
 
+      // disable right-click (context-menu) for purchased content
+      if(isPurchasedContent){
+        // disable right click
+        document.querySelector('video.vjs-tech').setAttribute('oncontextmenu', "return false;")
+      }
+
```

Comment:
- [ ] Do we want to disable the download button as well, or just let the backend error message handle it?  But if the backend is already preventing it, why bother preventing downloads from the context menu?
- [ ] If this is retained, maybe change the comment to "disable right-click (context menu) to prevent downloads" to explain the reason rather than what was change.
  - [ ] Also remove redundant `// disable right click` below it.
- [ ] Please ensure linter/prettier is ran to fix the spacing.

Reply:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/64950861/181496041-8c9b1763-7e34-4ac3-9017-9dde8327e96d.png">

Reply:
Oh never mind, saw the download button disabled.